Demo2.js

```diff
@@ -3,6 +3,12 @@
 import React from 'react';
 import Springs from './Springs';
 
+if (!String.prototype.includes) {
+  String.prototype.includes = function() {'use strict';
+    return String.prototype.indexOf.apply(this, arguments) !== -1;
+  };
+}
+
```

Comment:
ah. Does this repo need them enabled then? It was definitely throwing an error for me.